src/components/SearchBar.tsx

```diff
@@ -1,27 +1,26 @@
 'use client';
-import { ReactElement, FC, useState, useEffect } from 'react';
-// import { useRouter } from 'next/router';
+
+import { useState, useEffect } from 'react';
 import { useRouter } from 'next/navigation';
 import Link from 'next/link';
 import Image from 'next/image';
 import { initializeSearch } from 'utils/mapHelper';
-import { ILocation } from '../../types/types';
+// import { ILocation } from '../../types/types';
 
-const SearchBar: FC = (): ReactElement => {
-  const [location, setLocation] = useState<string | ILocation>('');
-  const router = useRouter()
+const SearchBar = () => {
+  // const [location, setLocation] = useState<string | ILocation>('');
+  const router = useRouter();
 
   useEffect(() => {
-    if (typeof window !== 'undefined') {
-      initializeSearch()
-        .then((selected) => {
-          setLocation(selected as ILocation);
-        })
-        .catch((error) => {
-          console.error(error);
-        });
-    }
-  }, [location]);
+    if (typeof window === 'undefined') return;
```

Comment:
remove comments?

Reply:
unnecessary code atm. so keeping it blanked until its used